Apps/WebClient/src/ClientApp/src/components/modal/verifySMS.vue

```diff
@@ -146,6 +166,18 @@ export default class VerifySMSComponent extends Vue {
         });
     }
 
+    private getTimedOut(): boolean {
+        if (
```

Comment:
this can be just 
```
return moment()
                .subtract(this.config.timeouts!.resendSMS, "minutes")
                .isBefore(moment(this.SMSResendDateTime));
```

Reply:
Done